### DISCUSSION TITLE
Ideias Skyten

### DISCUSSION THREAD

**Discussion by Batatoones** [leogianfagna/IssueTracker#928] [Ideia]
### Servidor afetado

Skyten

### Qual o propósito da ideia?

Vendas + Agradar público

### Descrição da ideia

Por que não adicionar efeitos visuais no skyten como possuí no lobby? Poderia ser vendido em pacotes baratos e requisitaram cosméticos pois gostaram de me ver usando alguns. 
Acho que da mesma maneira que possui cosméticos no henesys e quando vai pra X local o cosmético some pra evitar poluição visual, poderia ser feito no Skyten como influência de venda + agradar o público.

**Comment by leogianfagna** [comment]
@Batatoones cosméticos + efeitos visuais quebraria muito a otimização do servidor

**Comment by Batatoones** [comment]
@leogianfagna e se fosse um por vez? ou usa cosmético ou usa efeito visual

**Comment by leogianfagna** [comment]
@Batatoones podemos testar (e na pratica ver se da pra bloquear os dois juntos) e iniciar como teste falando que pode ser removido a qualquer momento, ai ja vemos a aprovação da galera se é positiva, negativa ou insignificante

mas só vai dar pra colocar as particulas mesmo pois os demais cosmeticos é bugadasso e não caem bem pra um servidor mais free (daria pra colocar efeitos de morte e de flecha tambem)

**Comment by Batatoones** [comment]
@leogianfagna A ideia é partículas e foi sugestão de jogador pra ser aplicado. Efeitos de mortes seria algo bom no vanillew que possuí mais pvp quem sabe? Acho que poderia ser aplicado em ambos pois poderia ser um conteúdo bom

**Comment by leogianfagna** [comment]
![image](https://github.com/user-attachments/assets/929cd87f-4afa-4d06-8c59-146d3629b27d)
